docs/tutorial/ecommerce-tutorial/index.md

```diff
@@ -102,6 +102,18 @@ You have 2 keys in both test mode and production mode:
 
 While testing, you must use the key(s) that include _test_. For production code, you will need to use the live keys. As the names imply, your publishable key may be included in code that you share publicly (for example, on the frontend, and in GitHub), whereas your secret key should not be shared with anyone or committed to any public repo. It’s important to restrict access to this secret key because anyone who has it could potentially read or send requests from your Stripe account and see information about charges or purchases or even refund customers.
 
+### Enabling the "Checkout client-only integration" for your Stripe account
+
+Through this tutorial you will be using the "Checkout client-only integration" from Stripe. To use this integration you need to activate it on the corresponding [Checkout settings](https://dashboard.stripe.com/account/checkout/settings) from your Stripe Dashboard.
+
+![Stripe control to enable the Checkout client-side only integration highlighted](stripe-checkout-clientside-functionality.png)
+
+> 💡 This change will also modify the interface that Stripe provides to administrate your products, keep this in mind in case you have previously used this tool. If you have never used the product administrator, you don't need to worry.
```

Comment:
```suggestion
> 💡 This change will also modify the interface that Stripe provides to administer your products: keep this in mind in case you have previously used this tool. If you have never used the product administrator, you don't need to worry.
```